Mathlib/RingTheory/LocalProperties/Projective.lean

```diff
@@ -0,0 +1,114 @@
+/-
+Copyright (c) 2024 Andrew Yang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Andrew Yang
```

Comment:
David Swinarski is listed as Co-Author of the PR, maybe then also add them here?